src/modules/RelocationServer.rb

```diff
@@ -245,24 +257,28 @@ def ReadLibvirtServices
       @libvirtd_enabled = Service.Enabled("libvirtd")
       @sshd_enabled = Service.Enabled("sshd")
 
-      if Service.Status("libvirtd") == 0
+      if Service.active?("libvirtd") == 0
         @libvirtd_is_running = true
         Builtins.y2milestone("libvirtd is running")
       else
         @libvirtd_is_running = false
         Builtins.y2milestone("libvirtd is not running")
       end
-      if Service.Status("sshd") == 0
+      if Service.active?("sshd") == 0
```

Comment:
active? method returns a boolean, so the comparison with 0 is not valid anymore.